sdk/communication/azure-communication-chat/README.md

```diff
@@ -79,11 +81,11 @@ AzureCommunicationChatServiceAsyncClient client = new AzureCommunicationChatServ
 
 User access tokens enable you to build client applications that directly authenticate to Azure Communication Services. Refer [here](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) to learn how to create a user and issue a User Access Token.
 
-The id for the user created above will be necessary later to add said user as a member of a new chat thread. The initiator of the create request must be in the list of members of the chat thread.
+The id for the user created above will be necessary later to add said user as a participant of a new chat thread. The initiator of the create request must be in the list of participants of the chat thread.
```

Comment:
This part is not true anymore : 'The initiator of the create request must be in the list of participants of the chat thread.'
Lets start removing that from comments and docs with the changes we are doing.